packages/lib/src/link/Link.tsx

```diff
@@ -107,7 +107,7 @@ const StyledLink = styled.div<{
        cursor: pointer;`}
   }
   &:focus {
-    border-radius: 2px;
+    border-radius: 4px;
```

Comment:
If you hover the link while being focused, the `border-bottom` bends over the edges, which was noticeable until now, since the `border-radius` has been increased, we need to figure out another way to implement this.

Reply:
I'll contact design to determine what the best solution for this is 
![image](https://github.com/user-attachments/assets/a1bdfea2-4e2f-4d2b-8b4c-ca7258bdaf68)

Reply:
After talking to Irene, adding a 2px padding and slightly changing how the styles are applied seems to be appropiate.